src/app.js

```diff
@@ -0,0 +1,75 @@
+require("babel-runtime/regenerator");
+require("webpack-hot-middleware/client?reload=true");
+require('./favicon.ico');
+
+require("./scss/cart.scss");
```

Comment:
Why do you need all the css here ?

Reply:
Rahul, I had done this in two ways,
1. to include these scss in entry property webpack config file.
2. require files in main entry file of webpack config file.
I prefer all scss files in main entry file of webpack config.

Please suggest me some other ways doing this.